packages/frontend/src/components/Swap/SwapAndSuccessContainer.js

```diff
@@ -0,0 +1,174 @@
+        margin: 0 auto;
+        width: fit-content;
+        margin-bottom: 30px;
+        display: flex;
+        justify-content: center;
+        align-items: center;
+        padding: 8px 8px 5px 5px;
+        border-radius:50%;
+        border: 1px solid #3170c7;
+        :hover {
+            svg {
+                g, path {
+                stroke: #0072ce;
+                fill: #0072ce;
+             }
+        }
+    }
+
+        svg {
+            transform: rotate(90deg);
+            cursor: pointer;
+            g:hover {
+                stroke: #0072ce;
+                fill: #0072ce;
+             }
+        }
+    }
+
+    .buttons-bottom-buttons {
+        margin-top: 30px;
+
+        > button {
+            display: block;
+            width: 100%;
+        }
+
+        .link {
+            display: block !important;
+            margin: 20px auto !important;
+        }
+    }
+
+    .text_info_success {
+        width: fit-content;
+        font-style: normal;
+        font-weight: 700;
+        font-size: 20px;
+        line-height: 28px;
+        text-align: center;
+        color: #252729;
+        margin: 0 auto;
+    }
+`;
+
+
+const SwapAndSuccessContainer = ({
+    fungibleTokensList,
+    accountId,
+    multiplier,
+}) => {
+    const [from, setFrom] = useState(fungibleTokensList[0]);
+    const [to, setTo] = useState(currentToken(fungibleTokensList, 'USN'));
+    const [inputValueFrom, setInputValueFrom] = useState(0);
+    const swapContractValue = useSelector(selectSwapBycontractName)
+    const [activeView, setActiveView] = useState(VIEWS_SWAP.MAIN);
+    const dispatch = useDispatch()
+
+    useEffect(() => {
+        if(swapContractValue && swapContractValue === 'NEAR') {
+            setFrom(currentToken(fungibleTokensList, 'USN'));
+            setTo(fungibleTokensList[0]);
+            return;
+        } 
+
+        setFrom(currentToken(fungibleTokensList, from?.onChainFTMetadata?.symbol));
+        setTo(currentToken(fungibleTokensList, to?.onChainFTMetadata?.symbol || 'USN'));
+        
+    }, [fungibleTokensList]);
+
+
+    useEffect(() => {
+        return () => dispatch(handleSwapBycontractName(''))
+    },[dispatch])
+
+    const onHandleSBackToSwap = useCallback(async () => {
+        await dispatch(refreshAccount())
+        await dispatch(fetchTokens({ accountId }));
+        setActiveView('main')
+    },[]) 
+
+   const getCurrentViewComponent = (activeView) => {
+    switch (activeView) {
+    case VIEWS_SWAP.MAIN:
+            return (
+                <SwapPage
+                    setActiveView={setActiveView}
+                    accountId={accountId}
+                    from={from}
+                    inputValueFrom={inputValueFrom}
+                    multiplier={multiplier}
+                    setInputValueFrom={setInputValueFrom}
+                    to={to}
+                    onSwap={() => {
+                        if (to?.balance === '0' || !to?.balance) return;
```

Comment:
This functionality does not suit us, since it depends on the value from the input, and it is initially zero. Here the meaning is that if the lower token (which is "to") has a zero balance, then the user will not be able to swap it up (which is "from"). Because it is impossible to swap from a zero balance.